server/routerlicious/packages/lambdas/src/deli/lambdaFactory.ts

```diff
@@ -180,8 +182,36 @@ export class DeliLambdaFactory extends EventEmitter implements IPartitionLambdaF
 					Lumberjack.info(message, getLumberBaseProperties(documentId, tenantId));
 				}
 			} else {
-				lastCheckpoint = JSON.parse(document.deli);
-			}
+                let checkpoint;
+                let isLocalCheckpoint = false;
+                if(this.localCheckpointEnabled){
+                    // Search local db for checkpoint
+                    Lumberjack.info(`Searching local collection for checkpoint.`, getLumberBaseProperties(documentId, tenantId));
```

Comment:
Also maybe export find last checkpoint as a common function? So consumer in FRS could use it?